cmd/access/staked_access_node_builder.go

```diff
@@ -32,3 +48,48 @@ func (sanb *StakedAccessNodeBuilder) Initialize() cmd.NodeBuilder {
 
 	return sanb
 }
+
+// enqueueUnstakedNetworkInit enqueues the unstaked network component initialized for the staked node
+func (sanb *StakedAccessNodeBuilder) enqueueUnstakedNetworkInit() {
+
+	sanb.Component("unstaked network", func(node cmd.NodeBuilder) (module.ReadyDoneAware, error) {
+
+		// NodeID for the staked node on the unstaked network
+		unstakedNodeID := sanb.NodeID() // currently set the same as the staked NodeID
+
+		// Networking key
+		unstakedNetworkKey := sanb.NetworkKey()
```

Comment:
Nit: Because the public identity (at least key, and maybe ID as well) should not be the same, we'll want to leave a TODO here.